stdlib/public/Concurrency/TaskGroup.cpp

```diff
@@ -839,13 +850,16 @@ static void swift_taskGroup_cancelAllImpl(TaskGroup *group) {
 }
 
 bool TaskGroupImpl::cancelAll() {
+  SWIFT_TASK_DEBUG_LOG("cancel all tasks in group = %p", this);
+
   // store the cancelled bit
   auto old = statusCancel();
   if (old.isCancelled()) {
     // already was cancelled previously, nothing to do?
     return false;
   }
 
+  // FIXME: must also remove the records!!!!
```

Comment:
Is this still TODO?

Reply:
Uff I think so... sorry, ran out of weekend :-\ 

I'm continuing to look at this and the other reported leak...